config/manager/controller-config.yaml

```diff
@@ -19,3 +19,4 @@ data:
         - kind: InterstepBufferService
           schedule: "0,0,10"
       analysisRunTimeout: 1200
+    permittedRiders: "group=autoscaling.k8s.io,kind=VerticalPodAutoscaler"
```

Comment:
I think we are likely to deploy VerticalPodAutoscaler, so I ended up deciding to add it here after all. (Not that it matters too much, since we override this in NMG anyway)